py/mkenv.mk

```diff
@@ -59,7 +59,14 @@ LD += -m32
 endif
 
 MAKE_FROZEN = $(TOP)/tools/make-frozen.py
+# allow mpy-cross (for WSL) and mpy-cross.exe (for cygwin) to coexist
+ifeq ($(OS),Windows_NT)
+MPY_CROSS = $(TOP)/mpy-cross/mpy-cross.exe
```

Comment:
This line should not be needed.

Reply:
I am testing Cygwin and Linux (WSL) builds on the same working copy. I have mpy-cross and mpy-cross.exe in the same directory. This is the easiest way to verify changes under both build environments.
Without this line Cygwin fails because it tries to execute the Linux version
`MPY modules/empty.py
../../mpy-cross/mpy-cross: ../../mpy-cross/mpy-cross: cannot execute binary file
make: *** [build-STM32L476DISC/frozen_mpy/empty.mpy] Error 126
`

Reply:
ok